src/modules/day/calendar-day-view.scss

```diff
@@ -48,14 +48,20 @@
   }
 
   .cal-event {
-    border: solid 1px;
     padding: 5px;
     font-size: 12px;
     overflow: hidden;
     text-overflow: ellipsis;
     white-space: nowrap;
     height: 100%;
     box-sizing: border-box;
+    background-color: #D1E8FF;
+    border: 1px solid #1e90ff;
+    color: #1e90ff;
+  }
+
+  .cal-event-title:link {
+    color: currentColor;
```

Comment:
sorry for this additional change. I can also remove it again but it was strange to see dots from ellipsis in the specified color and the link in blue (bootstrap) in the demos..

Reply:
I learnt something new today from this, thanks! 😄